.github/workflows/build-check.yml

```diff
@@ -0,0 +1,37 @@
+name: Check Build Status
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "agenta-web/**"  
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+
+      - name: Install dependencies
+        run: npm ci
```

Comment:
let's use --omit=dev like how we use here
https://github.com/Agenta-AI/agenta/blob/main/agenta-web/prod.Dockerfile#L7

Reply:
> also isn't npm run ci not npm ci?

there's no script defined for `npm run ci` in `package.json` inside agenta-web. so i think it wouldn't work.